tools/payloads/configbytes/apk_configbytes_extract.rb

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
```

Comment:
This is failing on a lib load error for me, adding the following here addresses it be adjusting the load path. This is how the other tools work.

```suggestion
msfbase = __FILE__
while File.symlink?(msfbase)
  msfbase = File.expand_path(File.readlink(msfbase), File.dirname(msfbase))
end
$:.unshift(File.expand_path(File.join(File.dirname(msfbase), '..', '..', '..', 'lib')))
```